### DISCUSSION TITLE
【图集】美图分享21

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1425] [其他]
大文件测试，请准备个好的VPN

感谢@Buer-Nahida 提供的方法。目前可以发布大于10MB的小图片。

其余的图片看我的仓库：[Github@ImageStorage_AD4F266F - 图片存储仓库AD4F266F](https://github.com/capacitor1/ImageStorage_AD4F266F/tree/master)

![赠Buer-Nahida一张图](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random931.jpg)

![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1023.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1170.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1256.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1288.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1323.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1407.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random1409.jpg)

![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random926.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random914.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random905.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random888.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random572.jpg)
![](https://raw.githubusercontent.com/capacitor1/ImageStorage_AD4F266F/master/Random579.jpg)

**Comment by Buer-Nahida** [comment]
挺感谢楼主，吃井不忘挖水人，居然@我了一下

**Reply by Buer-Nahida** [reply]
不对，是吃水不忘挖井人

**Comment by kallen1314** [comment]
啊，有崩2的卡琳！
